webpack.config.prod.js

```diff
@@ -39,6 +39,15 @@ module.exports = {
           path.join(__dirname, "playground"),
           path.join(__dirname, "node_modules"),
         ],
+      },
+      {
+        test: /\.json$/,
+        loader:"json-loader",
+        include: [
+          path.join(__dirname, "css"),
+          path.join(__dirname, "playground"),
+          path.join(__dirname, "node_modules"),
+        ],
```

Comment:
What's up with these json-loader changes? Do we import JSON explicitly anywhere in the project? I don't see any new uses of this functionality as part of this PR..

Reply:
Ajv loads two json files so I added the loader for that.

ajv/lib/refs/json-schema-draft-06.json
ajv/lib/refs/$data.json

It uses draft 6 by default but we can specify an earlier revision if needed. Not sure if using v6 would be a breaking change for this lib.

Reply:
Ah, great, thanks. I think draft 6 is what we target at the moment.